molecule/default/molecule.yml

```diff
@@ -9,64 +9,24 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - name: amazonlinux2_systemd
-    image: geerlingguy/docker-amazonlinux2-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
-  - name: debian10_systemd
-    image: geerlingguy/docker-debian10-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
+  - name: amazonlinux2
+    image: amazonlinux:2
+  - name: debian10
+    image: debian:buster-slim
   # The freeipa-client package does not appear to be present in
   # Debian testing right now.
-  # - name: debian11_systemd
-  #   image: cisagov/docker-debian11-ansible:latest
-  #   privileged: yes
-  #   volumes:
-  #     - /sys/fs/cgroup:/sys/fs/cgroup:ro
-  #   command: /lib/systemd/systemd
-  #   pre_build_image: yes
-  # - name: kali_systemd
-  #   image: cisagov/docker-kali-ansible:latest
-  #   privileged: yes
-  #   volumes:
-  #     - /sys/fs/cgroup:/sys/fs/cgroup:ro
-  #   command: /lib/systemd/systemd
-  #   pre_build_image: yes
-  - name: fedora32_systemd
-    image: geerlingguy/docker-fedora32-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
-  - name: fedora33_systemd
-    image: cisagov/docker-fedora33-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
-  - name: ubuntu_bionic_systemd
-    image: geerlingguy/docker-ubuntu1604-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
-  - name: ubuntu_xenial_systemd
-    image: geerlingguy/docker-ubuntu1804-ansible:latest
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:ro
-    command: /lib/systemd/systemd
-    pre_build_image: yes
+  # - name: debian11
+  #   image: debian:bullseye-slim
+  # - name: kali
+  #   image: kalilinux/kali-rolling
```

Comment:
What's up with `debian11` and `kali`?

Reply:
I'm not sure.  [Debian 11 (Bullseye, the current Debian testing) does not include the package right now, although Debian 12 (Sid, the current Debian unstable) does](https://packages.debian.org/search?keywords=freeipa).  Kali is always based on Debian testing, so it does not have the package either.

I assume this is an oversight that will be corrected prior to Debian Bullseye becoming the new Debian stable.